classes/range.js

```diff
@@ -104,7 +104,7 @@ class Range {
     range = range.replace(re[t.CARETTRIM], caretTrimReplace)
 
     // normalize spaces
-    range = range.split(/\s+/).join(' ')
+    range = range.replace(/  +/g, ' ')
```

Comment:
this change means it will no longer select non-space whitespace characters, including tabs and other unicode whitespace chars.

Reply:
you should provide tests to protect it

Reply:
Indeed, there should be tests, but few projects are willing to change behavior just because there’s not a test yet.